jenkins/Jenkinsfile

```diff
@@ -63,29 +63,15 @@ pipeline {
             sh script: "mkdir -p ${env.WORKSPACE}/py3_compat && cp -rf ${env.WORKSPACE}/${env.PROJECT}/. ${env.WORKSPACE}/py3_compat", label: "copying repo to seperate workspace"
             container('py27'){
               dir("${env.WORKSPACE}/py3_compat"){
-                echo 'install futurize, remove compat modules and find python3-incompatible code'
+                echo 'Prepare for py3compat'
```

Comment:
you can just use a `label` for the `sh`

Reply:
True but I forget about it sometimes. I know it doesn't do any actual change in the performance so I'll skip it this time, but will add it next time